cmd/kubeadm/app/cmd/upgrade/apply.go

```diff
@@ -88,7 +80,6 @@ func NewCmdApply(apf *applyPlanFlags) *cobra.Command {
 	cmd.Flags().BoolVar(&flags.dryRun, options.DryRun, flags.dryRun, "Do not change any state, just output what actions would be performed.")
 	cmd.Flags().BoolVar(&flags.etcdUpgrade, "etcd-upgrade", flags.etcdUpgrade, "Perform the upgrade of etcd.")
 	cmd.Flags().BoolVar(&flags.renewCerts, options.CertificateRenewal, flags.renewCerts, "Perform the renewal of certificates used by component changed during upgrades.")
-	cmd.Flags().DurationVar(&flags.imagePullTimeout, "image-pull-timeout", flags.imagePullTimeout, "The maximum amount of time to wait for the control plane pods to be downloaded.")
```

Comment:
I think this should go through deprecation...

Reply:
got it

Reply:
I am not sure if we should keep this flag operational until we can remove it.
@neolit123 @fabriziopandini WDYT?

Reply:
technically the flag should be operational until we remove it, which might be a bit difficult given we are removing the backend that it targets.